File: src/main/java/com/rs/java/game/area/banks/MobilisingArmiesBank.java
package com.rs.java.game.area.banks;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.Shape;
import com.rs.java.game.area.shapes.Polygon;

public final class MobilisingArmiesBank extends Area {

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Mobilising Armies Bank";
	}

	@Override
	public Shape[] shapes() {
		return new Shape[] { new Polygon(new WorldTile[] {
				new WorldTile(2440, 2840, 0),
				new WorldTile(2440, 2843, 0),
				new WorldTile(2405, 2843, 0),
				new WorldTile(2405, 2840, 0),
				new WorldTile(2440, 2840, 0),
		}) };
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.SAFEZONE;
	}

}

File: src/main/java/com/rs/kotlin/game/world/area/areas/banks/MobilisingArmiesBank.kt
package com.rs.kotlin.game.world.area.areas.banks

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Polygon
class MobilisingArmiesBank : Area(
    Polygon(
        arrayOf(
            WorldTile(2440, 2840, 0),
            WorldTile(2440, 2843, 0),
            WorldTile(2405, 2843, 0),
            WorldTile(2405, 2840, 0),
            WorldTile(2440, 2840, 0),
        )
    )
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Mobilising Armies Bank"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.SAFEZONE
    }
}
